Device: cuda
Traceback (most recent call last):
  File "/home/students/Photonics_CST/main.py", line 45, in <module>
    model=train(model,optimizer,trainloader,valloader,criterion,device,epochs=epochs)
  File "/home/students/Photonics_CST/util.py", line 17, in train
    for i,(x, y) in enumerate(trainloader):
  File "/home/students/miniconda3/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 345, in __next__
    data = self._next_data()
  File "/home/students/miniconda3/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 385, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/students/miniconda3/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/students/miniconda3/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/students/Photonics_CST/torchData.py", line 24, in __getitem__
    x = self.transform(x)
  File "/home/students/miniconda3/lib/python3.7/site-packages/torchvision/transforms/transforms.py", line 70, in __call__
    img = t(img)
  File "/home/students/miniconda3/lib/python3.7/site-packages/torchvision/transforms/transforms.py", line 101, in __call__
    return F.to_tensor(pic)
  File "/home/students/miniconda3/lib/python3.7/site-packages/torchvision/transforms/functional.py", line 55, in to_tensor
    raise TypeError('pic should be PIL Image or ndarray. Got {}'.format(type(pic)))
TypeError: pic should be PIL Image or ndarray. Got <class 'torch.Tensor'>
